shotgun_api3/lib/httplib2/python3/socks.py

```diff
@@ -434,7 +434,7 @@ def __negotiatehttp(self, destaddr, destport):
         wrote_host_header = False
         wrote_auth_header = False
         if self.__proxy[6] != None:
-            for key, val in self.__proxy[6].iteritems():
+            for key, val in self.__proxy[6].items():
```

Comment:
Similar change can be done in `../python2/socks.py`, intentionally left that one as-is (as it's still syntactically OK, and presumably this one is the one selected at runtime for py3)